src/firewall-applet.in

```diff
@@ -58,14 +58,19 @@ def search_app(app):
     return None
 
 NM_CONNECTION_EDITOR = ""
-for binary in [ "/usr/bin/nm-connection-editor",
+for binary in [ "/usr/bin/systemsettings",
+                    "/bin/systemsettings",
+                "/usr/bin/nm-connection-editor",
                     "/bin/nm-connection-editor",
                 "/usr/bin/kde5-nm-connection-editor",
                     "/bin/kde5-nm-connection-editor",
                 "/usr/bin/kde-nm-connection-editor",
                     "/bin/kde-nm-connection-editor" ]:
     if os.path.exists(binary):
-        NM_CONNECTION_EDITOR = binary
+        if binary.endswith("systemsettings"):
+            NM_CONNECTION_EDITOR = binary + " kcm_networkmanagement"
```

Comment:
Why append "kcm_networkmanagement" here instead of in the array entry?

Reply:
Changed, done!